docs/section12/Rule12-2.md

```diff
@@ -1,36 +1,40 @@
-
-# Receptacle - Rule 12-2
-
+# Receptacles - Rule 12-2
 **Rule ID:** 12-2  
-**Rule Description:** Number of spaces modeled in User RMR and Proposed RMR are the same  
-**Rule Assertion:** Proposed RMR = User RMR  
+**Rule Description:** Based on the space type, receptacle controls may be required by Section 8.4.2. Receptacle schedules shall be identical to the proposed design except when receptacle controls, installed in spaces where not required by Section 8.4.2, are included in the proposed building design.  
```

Comment:
Recommended edit:
"Depending on the space type, receptacle controls may be required by 90.1 Section 8.4.2. Receptacle schedules shall be modeled identically to the proposed design except when receptacle controls are specified in the proposed design for spaces where not required by Section 8.4.2."